languages/typescript/dockerfiles/bun-1.1.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM oven/bun:1.1-alpine
+
+ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="package.json,bun.lockb"
+
+WORKDIR /app
+
+COPY package.json ./
+COPY bun.lockb ./
```

Comment:
Let's use repository-aware caching instead: 

```
# .git & README.md are unique per-repository. We ignore them on first copy to prevent cache misses
COPY --exclude=.git --exclude=README.md . /app
```

(Check other dockerfiles in this repo for examples)